README.md

```diff
@@ -107,6 +109,8 @@ all the operations defined by the `sql:Client` will be supported by the `jdbc:Cl
 All database libraries share the same connection pooling concept and there are three possible scenarios for
 connection pool handling. For its properties and possible values, see the [`sql:ConnectionPool`](https://docs.central.ballerina.io/ballerina/sql/latest/records/ConnectionPool).
 
+>**Tip**: Connection pooling is used to optimize opening and closing connections to the database. However, the pool comes with an overhead. It is best to configure the connection pool properties as per the application need to get the best performance.
```

Comment:
Change this to note. I pushed some changes on the final PR, update all others based on the final changes